src/core/core.js

```diff
@@ -3,22 +3,9 @@
 module.exports = function() {
 
 	// Occupy the global variable of Chart, and create a simple base class
-	var Chart = function(context, config) {
-		var me = this;
-
-		// Support a jQuery'd canvas element
-		if (context.length && context[0].getContext) {
-			context = context[0];
-		}
-
-		// Support a canvas domnode
-		if (context.getContext) {
-			context = context.getContext('2d');
-		}
-
-		me.controller = new Chart.Controller(context, config, me);
-
-		return me.controller;
+	var Chart = function(item, config) {
```

Comment:
I wonder if `Chart.Controller` can eventually just become the `Chart` variable. #2481 asked about this and now that everything is in the controller, we might be able to do it.

Only issue I see is with any dependencies that the controller needs but we need the `Chart` variable to be defined first.